locale/cs.po

```diff
@@ -0,0 +1,131 @@
+# Czech translation of gnome-shell-extension-appindicator.
+# Copyright (C) 2022
+# This file is distributed under the same license as the AppIndicatorExtension package.
+# Daniel Rusek <mail@asciiwolf.com>, 2022.
+#
+msgid ""
+msgstr ""
+"Project-Id-Version: \n"
+"Report-Msgid-Bugs-To: \n"
+"POT-Creation-Date: 2021-09-27 20:43+0200\n"
+"PO-Revision-Date: 2022-04-09 00:16+0200\n"
+"Last-Translator: Daniel Rusek <mail@asciiwolf.com>\n"
+"Language-Team: \n"
+"Language: cs\n"
+"MIME-Version: 1.0\n"
+"Content-Type: text/plain; charset=UTF-8\n"
+"Content-Transfer-Encoding: 8bit\n"
+"Plural-Forms: nplurals=3; plural=((n==1) ? 0 : (n>=2 && n<=4) ? 1 : 2);\n"
+"X-Generator: Poedit 3.0.1\n"
+
+#: prefs.js:67
+msgid "Opacity (min: 0, max: 255)"
+msgstr "Krytí (min: 0, max: 255)"
```

Comment:
Doesn't „průhlednost“ sound better?

Reply:
If I am not mistaken, "opacity" means the opposite of transparency, therefore "krytí" is more accurate translation in this case.